test/test.js

```diff
@@ -761,6 +761,52 @@ test( "elementValue() finds radios/checkboxes only within the current form", fun
 	ok( !v.elementValue( foreignRadio ) );
 } );
 
+test( "elementValue() returns the file input's name without the prefix 'C:\\fakepath\\' ", function() {
+	var v = $( "#userForm" ).validate(),
+		fileInput = {
+			name: "fakeFile",
+			type: "file",
+			files: {},
+			nodeName: "INPUT",
+			value: "C:\\fakepath\\somefile.txt",
+			form: $( "#userForm" )[ 0 ],
+			hasAttribute: function() { return false; }
+		},
+		oldRules = $.fn.rules,
+		oldvalidationTargetFor = v.validationTargetFor;
+
+	equal( v.elementValue( fileInput ), "somefile.txt" );
+
+	$.fn.rules = function() {
```

Comment:
I was getting strange behavior using the public api directly because I'm using a fake input not a real one, but right now, I get an idea to solve that. will push again.